2-Browsers/Week1/homework/ex6-gameOfLife/index.js

```diff
@@ -24,9 +21,23 @@ function draw(cell, context) {
     CELL_SIZE,
     CELL_SIZE
   );
-
   if (cell.alive) {
-    context.fillStyle = `rgb(24, 215, 236)`;
+    let opacity;
+    switch (cell.lifeTime) {
```

Comment:
This is 100% a stylistic choice, but if you're breaking on every case in a switch, I'd personally prefer an if..else statement.